src/mobile/android/app/build.gradle

```diff
@@ -152,6 +152,8 @@ android {
     }
 }
 dependencies {
+    compile project(':react-native-camera')
```

Comment:
This dependency is already declared here:
https://github.com/iotaledger/trinity-wallet/pull/781/files#diff-fea0f0b62d012000a34c988f704b5671R168